__tests__/app.test.js

```diff
@@ -98,3 +98,146 @@ describe("GET /api/articles", () => {
       });
   });
 });
+
+describe("GET /api/articles/:article_id/comments", () => {
+  test("responds with an array of comments for the given article_id", () => {
+    return request(app)
+      .get("/api/articles/5/comments")
+      .then(({ body }) => {
+        const { comments } = body;
+        expect(comments).toHaveLength(2);
+        comments.forEach((comment) => {
+          expect(comment).toMatchObject({
+            comment_id: expect.any(Number),
+            votes: expect.any(Number),
+            created_at: expect.any(String),
+            author: expect.any(String),
+            body: expect.any(String),
+            article_id: expect.any(Number),
+          });
+        });
+      });
+  });
+  test("responds with array in desc order", () => {
+    return request(app)
+      .get("/api/articles/5/comments")
+      .then(({ body }) => {
+        const { comments } = body;
+        expect(comments).toBeSorted({ descending: true });
+      });
+  });
+  test("GET:400, responds with status code and message when given invalid id", () => {
+    return request(app).get("/api/articles/wrong/comments").expect(400);
+  });
+  test("GET: 200, responds with empty array when artist_id is valid but no comments exists", () => {
+    return request(app)
+      .get("/api/articles/10/comments")
+      .expect(200)
+      .then(({ body }) => {
+        const { comments } = body;
+        expect(comments).toHaveLength(0);
+      });
+  });
+});
+
+describe("POST /api/articles/:article_id/comments", () => {
+  test("Inserts the new comment and then responds with the posted comment", () => {
```

Comment:
the convention you followed for test descriptions in the previous tests is better ie. POST:201